backend/app/data/test/wikipedia_arkg.ttl

```diff
@@ -1,25 +1,36 @@
-@prefix urn: <http://imlapps.github.io/anti-recommender/anti-recommendation/> .
-
 @prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
 @prefix schema: <http://schema.org/> .
+@prefix urn: <http://imlapps.github.io/anti-recommender/anti-recommendation/> .
 @prefix wd: <http://www.wikidata.org/entity/> .
+@prefix wikibase: <http://wikiba.se/ontology#> .
 
 urn:uuid:edf5a5c1-014e-4dfa-a503-8cbf1672aac8 rdf:type schema:Recommendation .
```

Comment:
Nit: this doesn't seems quite right, with schema:about and schema:itemReviewed. I think schema:itemReviewed is probably the original record. It's not clear what the "next record" should be.

Leave it for now.